pyxem/generators/integration_generator.py

```diff
@@ -164,12 +163,11 @@ def _get_intensities_summation_method(z,
                   f"I: {inty:.2f} | Sigma(I): {sigma:.2f} | SNR(I): {snr:.2f} | I/pix: {inty/n_pix:.2f} \n"
                   f"i: {i:.2f} | j: {j:.2f} | dX: {dX:.2f} | dY: {dY:.2f} | X: {X:.2f} | Y: {Y:.2f} ")
             # for debugging purposes
-            plot = True
-            if plot:
-                plt.imshow(signal)
-                plt.plot(dY + box_inner, dX + box_inner, "r+")  # center_of_mass
-                plt.plot(box_inner, box_inner, "g+")        # input
-                plt.show()
+            import matploltib.pyplot as plt
+            plt.imshow(signal)
+            plt.plot(dY + box_inner, dX + box_inner, "r+")  # center_of_mass
+            plt.plot(box_inner, box_inner, "g+")        # input
+            plt.show()
```

Comment:
Why did you remove the if statement here?

Reply:
Because I find the following a bit silly and not necessary:
```python
plot = True
if plot:
   ...
```

Reply:
Oh, yes, that was silly - thanks!